shared/wallets/wallet/index.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import * as React from 'react'
+import {Box2, ClickableBox, Icon, Text, Avatar, FloatingMenu} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../../styles'
+import {FloatingMenuParentHOC, type FloatingMenuParentProps} from '../../common-adapters/floating-menu'
+
+type Props = {
+  isSelected: boolean,
+  name: string,
+  keybaseUser: string,
+  contents: string,
+  onSelect: () => void,
+}
+
+class Wallet extends React.PureComponent<Props> {
+  render() {
+    const color = this.props.isSelected ? globalColors.blue : globalColors.white
+
+    const titleStyle = {
+      ...globalStyles.fontSemibold,
+      color: this.props.isSelected ? globalColors.white : globalColors.darkBlue,
+      backgroundColor: color,
+      fontSize: 13,
+    }
+    const amountStyle = {
+      color: this.props.isSelected ? globalColors.white : globalColors.black_40,
+      backgroundColor: color,
+      fontSize: 11,
+    }
+    const props = this.props
+    return (
+      <ClickableBox onClick={props.onSelect} style={{backgroundColor: color}}>
+        <Box2 style={{height: rowHeight, backgroundColor: color}} direction="horizontal" fullWidth={true}>
+          <Box2 direction="horizontal">
+            <Icon
+              type={'iconfont-hand-wave'}
+              color={globalColors.darkBlue}
+              fontSize={28}
+              style={{marginLeft: globalMargins.tiny, marginRight: globalMargins.tiny}}
+            />
+          </Box2>
+          <Box2 direction="vertical">
+            <Box2 direction="horizontal" fullWidth={true}>
+              {this.props.keybaseUser && (
+                <Avatar
+                  size={16}
+                  username={this.props.keybaseUser}
+                  style={{marginRight: globalMargins.xtiny}}
```

Comment:
box2 has a gap property whose purpose is to remove all these margins inside of components to space them